sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -265,65 +262,13 @@ export class CertificateClient {
       },
     };
 
-    this.client = new KeyVaultClient(
-      clientOptions.serviceVersion || LATEST_API_VERSION,
-      internalClientPipelineOptions,
-    );
+    this.client = new KeyVaultClient(this.vaultUrl, credential, {
```

Comment:
`vaultUrl` (the endpoint) has been promoted to the constructor level in the generated code instead of needing to be passed per operation.